web-app/pages/certificate/[tokenId].tsx

```diff
@@ -62,5 +78,27 @@ const Certificate: NextPage = () => {
     </Layout>
   );
 };
-
+export async function getServerSideProps(context: any) {
```

Comment:
@rashaabdulrazzak  I'm not sure what this function is for. Maybe this is where you were thinking of fetching the individual text details of the cert, and maybe here is where you meant was a "work in progress". 

If yes, I think we should just delete this function for now. I'd rather keep PRs small and quick.  It will be enough of a win if we can get the social share previews looking good even without the text details from on-chain.

Plus, there are efficiency concerns. I'd written as a future task for myself:

> Add cert text details to `web-app/pages/certificate/[tokenId].tsx` but consider efficiency; it should fetch certificate data just once, and then pass that to the image generator. fetchCertificateDetails should not be inside `web-app/pages/api/cert/[imageFileName].ts`

But maybe you can tackle it (instead of me) later as a future branch.

Reply:
sorry @ryancwalsh  I just saw your comments here 
if I delete it the title and description of all certifications will be the same for now is it fine?

Reply:
@rashaabdulrazzak I don't know what you mean.

If you mean:

```
<meta property="og:title" content="I got certified with NEAR university" />
        <meta property="og:description" content="View NEAR University certificates of any .near account" />
```
I thought those looked fine because those look generic and could make sense across all of our certification programs. 

Later, we can read on-chain data and change these fields to be more specific.